homeassistant/bootstrap.py

```diff
@@ -253,6 +248,7 @@ def from_config_dict(config: Dict[str, Any],
     if skip_pip:
         _LOGGER.warning('Skipping pip installation of required modules. '
                         'This may cause issues.')
+        pkg_util.mount_lib_path(hass.config.config_dir)
```

Comment:
At them moment it is done in a couple of places. The idea was to move it to `handle_requirements`, but skip_pip would be the exception (and require a specific mount)